Napzak-iOS/Napzak-iOS/Presentation/Mypage/MyPageView.swift

```diff
@@ -0,0 +1,170 @@
+                            .foregroundColor(Color.napzakGrayScale(.gray500))
+                    }
+                }
+            }
+            
+            Spacer()
+        }
+        .padding(20)
+        .background(Color.napzakGrayScale(.gray10))
+        .clipShape(RoundedRectangle(cornerRadius: 25))
+        .padding(.horizontal, 27)
+        .padding(.top, 123)
+    }
+    
+    private var marketButton: some View {
+        VStack(spacing: 0) {
+            Button {
+                // 내 마켓 보기
+            } label: {
+                HStack {
+                    Spacer()
+                    Text("내 마켓 보기")
+                        .font(.napzakFont(.caption1SemiBold12))
+                        .foregroundColor(Color.napzakGrayScale(.gray300))
+                    
+                    Image("arrow_right")
+                    Spacer()
+                }
+                .padding()
+                .background(Color.napzakGrayScale(.gray10))
+                .clipShape(RoundedRectangle(cornerRadius: 17))
+            }
+            .padding(.horizontal, 27)
+            .padding(.top, 20)
+            
+            Rectangle()
+                .fill(Color.napzakGrayScale(.gray10))
+                .frame(height: 4)
+                .padding(.top, 20)
+        }
+    }
+    
+    private var menuGrid: some View {
+        let menuItems: [(title: String, icon: String)] = [
+            ("판매 내역", "group1_icn"),
+            ("구매 내역", "group2_icn"),
+            ("최근 본 상품", "group3_icn"),
+            ("찜", "group4_icn"),
+            ("설정", "group5_icn"),
+            ("고객센터", "group6_icn")
+        ]
+        
+        let columns = Array(repeating: GridItem(.flexible(), spacing: 0), count: 3)
+        
+        return LazyVGrid(columns: columns, spacing: 0) {
+            ForEach(menuItems.indices, id: \.self) { index in
+                ZStack {
+                    menuItem(title: menuItems[index].title, iconName: menuItems[index].icon)
+                }
+                .frame(maxWidth: .infinity, minHeight: 81)
+                .background(Color.napzakGrayScale(.gray10))
+                .overlay(
+                    VStack {
+                        if index < 3 {
+                            Spacer()
+                            Rectangle()
+                                .fill(Color.napzakGrayScale(.gray50))
+                                .frame(height: 4)
+                        }
+                    }
+                )
+                .overlay(
+                    HStack {
+                        if (index + 1) % 3 != 0 {
+                            Spacer()
+                            Rectangle()
+                                .fill(Color.napzakGrayScale(.gray50))
+                                .frame(width: 4)
+                        }
+                    }
+                )
+            }
+        }
+        .background(Color.napzakGrayScale(.gray10))
```

Comment:
P3

전체 배경색을 설정해주고, 전체 Grid의 열간 간격(`GridItem`의 `spacing`), 행간 간격(`LazyVGrid`의 `spacing`)을 4로 설정해주면 좀 더 간단하게 코드를 작성할 수 있을 것 같아욧
```suggestion
        let columns = Array(repeating: GridItem(.flexible(), spacing: 4), count: 3)
        
        return LazyVGrid(columns: columns, spacing: 4) {
            ForEach(menuItems.indices, id: \.self) { index in
                ZStack {
                    menuItem(title: menuItems[index].title, iconName: menuItems[index].icon)
                }
                .frame(maxWidth: .infinity, minHeight: 82)
                .background(Color.napzakGrayScale(.gray10))
            }
        }
        .background(Color.napzakGrayScale(.gray50))
```